app/services/page_updater.rb

```diff
@@ -81,9 +81,10 @@ def refresh?
 
   def update_page
     return unless @params[:page]
+    @page.slug = nil
```

Comment:
What I wonder is whether this'd be desired behavior or if we should support slugs that reflect titles from the beginning. That might go wrong with automatic page-updates, though. In other words, if someone is in the process of modifying or changing the title, it'll always store a new slug that matches the new and incomplete title. We'd run out of valid slugs really fast.